lib/mock/elixir_auth_github.ex

```diff
@@ -0,0 +1,12 @@
+defmodule AuthMvp.Mock.ElixirAuthGithub do
```

Comment:
worth adding a `moduledoc` or `doc` comment to this file with a single line of explanation. 💭